### PR TITLE
cs fix

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: Route/CmsPageRouter.php [hunk 1]
```diff
@@ -337,6 +337,6 @@ protected function isPageAlias($name)
      */
     protected function isPageSlug($name)
     {
-        return (is_string($name) && $name == PageInterface::PAGE_ROUTE_CMS_NAME);
+        return is_string($name) && $name == PageInterface::PAGE_ROUTE_CMS_NAME;
```
Comment (OskarStark) [inline Route/CmsPageRouter.php]: @Soullivaneuh shall we use yoda here?

Comment (Soullivaneuh) [inline Route/CmsPageRouter.php]: Technically yes but you can do it on another PR.

Comment (OskarStark) [inline Route/CmsPageRouter.php]: is there a check for the php cs fixer?
